cli/cmd/host.go

```diff
@@ -286,6 +317,76 @@ func (c *ServicedCli) cmdHostAdd(ctx *cli.Context) {
 	c.outputDelegateKey(host, nat, privateKey, keyfileName, registerHost)
 }
 
+// serviced host add-private HOST:PORT POOLID [--memory SIZE|%] [--nat-address HOST:PORT]
+func (c *ServicedCli) cmdHostAddPrivate(ctx *cli.Context) {
```

Comment:
For this method if there is an error, please exit with a non-zero code.

Reply:
Done via `os.Exit(1)` in the body